src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -361,7 +361,7 @@ public async ValueTask<object> PeekOneAsync(CancellationToken cancellationToken)
         public bool TryPeekOne(out MessageEnvelope envelope, TimeSpan? max, CancellationToken cancellationToken)
         {
             var (success, result) = InternalTryPeekOneAsync(max, true, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.